autobatch/autobatch_test.go

```diff
@@ -70,6 +72,9 @@ func TestFlushing(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// flushing is async so we can rely on having it happening immediately
+	time.Sleep(100 * time.Millisecond)
```

Comment:
can we make these deterministic to avoid flaky tests?  (stub out time fns, inject mock clock. etc)

Reply:
Is there a library you would recommend for that?

Reply:
Recommend https://github.com/benbjohnson/clock